pyproject.toml

```diff
@@ -205,6 +204,10 @@ warn_return_any = true
 warn_unreachable = true
 warn_unused_ignores = true
 
+[[tool.mypy.overrides]]
```

Comment:
Are the double brackets intentionally? What is their purpose? Thanks.

Reply:
That is the syntax mypy wants:
https://github.com/python/mypy/blob/master/mypy/config_parser.py#L394